data/scripts.js

```diff
@@ -902,6 +902,11 @@ exports.BattleScripts = {
 		var baseTemplate = (template = this.getTemplate(template));
 		var name = template.name;
 
+		if (name === 'Castform') {
+			// If someone wants a Castform set, give it one of its subforme sets instead
+			template = this.getTemplate(this.random(2) ? 'castformsunny' : 'castformrainy');
```

Comment:
This is unnecessary. It skips Castform because it has no moves. The formes are obtainable by themselves as they now have moves.